src/backups.py

```diff
@@ -270,12 +272,13 @@ def _create_bucket_if_not_exists(self) -> None:
         except ValueError as e:
             logger.exception("Failed to create a session '%s' in region=%s.", bucket_name, region)
             raise e
-        bucket = s3.Bucket(bucket_name)
+        # Boto3 doesn't have typedefs
+        bucket = s3.Bucket(bucket_name)  # type: ignore
```

Comment:
There are community provided typing stubs like [boto3-stubs](https://pypi.org/project/boto3-stubs/) we can use in the future.